quickwit/quickwit-proto/protos/quickwit/router.proto

```diff
@@ -68,6 +68,7 @@ enum IngestFailureReason {
   INGEST_FAILURE_REASON_ROUTER_LOAD_SHEDDING = 8;
   INGEST_FAILURE_REASON_LOAD_SHEDDING = 9;
   INGEST_FAILURE_REASON_CIRCUIT_BREAKER = 10;
+  INGEST_FAILURE_REASON_UNAVAILABLE = 11;
```

Comment:
until all servers are upgraded, this will be converted to `IngestServiceError::Internal`